src/api/app/views/webui/user/index.html.haml

```diff
@@ -35,6 +35,6 @@
 
 :javascript
   $(function() {
-    if ("#{@users.count}" !== "0")
+    if ("#{h(@users.size)}" !== "0")
```

Comment:
Is to escape “unsafe” data, for "Cross-Site Scripting", `h` is an alias for `html_escape`